src/views/Home.vue

```diff
@@ -1,25 +1,36 @@
 <template>
   <div class="home">
-    <p id="logo" v-if="$route.meta.hideSmallLogo">
-      <router-link to="/">
-        <img
-          class="bigLogo"
-          alt="ELIXIR Beacon Network logo"
-          src="@/assets/beacon-network-logo.png"
-        />
-      </router-link>
-    </p>
-    <component
-      data-testid="component"
-      v-bind:is="componentName"
-      @changeSearchForm="toggleForm"
-    ></component>
+    <div class="home">
+      <p id="logo" v-if="$route.meta.hideSmallLogo">
+        <router-link to="/">
+          <img
+            class="bigLogo"
+            alt="ELIXIR Beacon Network logo"
+            src="@/assets/beacon-network-logo.png"
+          />
+        </router-link>
+      </p>
+
+      <component
+        data-testid="component"
+        v-bind:is="componentName"
+        @changeSearchForm="toggleForm"
+        @toggleListing="toggleListing"
+        @returnToPrevious="returnToPrevious"
+      ></component>
+      <b-switch v-model="toggleV2" id="v2Switch">
+        {{
+          toggleV2 ? "Beacon v2 search enabled" : "Beacon v2 search disabled"
+        }}
+      </b-switch>
```

Comment:
i think this is the sequence that you need to align with the search bar so best to do a separate div with its own set of csss properties to be able to style it